Add lesismal/nbio

lesismal opened the pull request:
Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standards) before sending pull request. Thanks!

**Please provide package links to:**

- github.com repo: https://github.com/lesismal/nbio
- pkg.go.dev: https://pkg.go.dev/github.com/lesismal/nbio?tab=doc
- goreportcard.com: https://goreportcard.com/report/github.com/lesismal/nbio
- coverage service link: https://codecov.io/gh/lesismal/nbio


Very good coverage

**Note**: that new categories can be added only when there are 3 packages or more.

**Make sure that you've checked the boxes below before you submit PR:**
- [x] I have added my package in alphabetical order.
- [x] I have an appropriate description with correct grammar.
- [x] I know that this package was not listed before.
- [x] I have added pkg.go.dev link to the repo and to my pull request.
- [x] I have added coverage service link to the repo and to my pull request.
- [x] I have added goreportcard link to the repo and to my pull request.
- [x] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard).

avelino commented:
Thank you for contributing with [awesome-go](https://github.com/avelino/awesome-go), we will revise your contribution as soon as possible.

Automation body links content check:
- godoc.org or pkg.go.dev: True
- goreportcard.com: True
- coverage: True

> your project is about to be approved, it's under revision, _it may take a few days_

lesismal commented:
Since Travis CI is always waiting in the task queue to be built, I have removed it and switched to Github-Actions:
https://github.com/lesismal/nbio/actions

I see the check for Travis CI was failed, do I need to add Travis CI to my project again?

avelino commented:
> > I see the check for Travis CI was failed, do I need to add Travis CI to my project again?

you don't need to add TravisCI to your project, Awesome-Go's TravisCI failed because it is not in alphabetical order

`    --- FAIL: TestAlpha/Libraries_for_working_with_various_layers_of_the_network. (0.00s)`

show more [here](https://travis-ci.com/github/avelino/awesome-go/builds/217096855)
